### DISCUSSION TITLE
Administrator account duplicated and not able to delete it.

### DISCUSSION THREAD

**Discussion by halwesit** [directus/directus#14879] [Archive]
### Describe the Bug

I have used Directus for a client, after a couple of months I noticed that my administrator account has been duplicated, and I could not able to delete it. also not able to change the role of the account.

![directus-1](https://user-images.githubusercontent.com/10852176/183075865-d694f2c9-814d-4bd9-8061-4f004fe6cb66.png)

### To Reproduce

Not able to reproduce it because I antecedently notice this bug.

### Errors Shown

when I want to delete it it will show me this error:
`{
  "errors": [
    {
      "message": "delete from \"directus_users\" where \"id\" in ($1) - update or delete on table \"directus_users\" violates foreign key constraint \"directus_files_uploaded_by_foreign\" on table \"directus_files\"",
      "extensions": {
        "code": "INTERNAL_SERVER_ERROR"
      }
    }
  ]
}`

### What version of Directus are you using?

9.14.1

### What version of Node.js are you using?

v16.14.2

### What database are you using?

Postgres 13

### What browser are you using?

Chrome, 

### How are you deploying Directus?

PaaS (Caprover) using docker image

**Comment by azrikahar** [comment]
Your current error seems like #9507, can you try un-linking the user avatar from the admin user you wish to remove, and try removing the user again?

With that said, I'm not sure what could possibly be the reason why there are 2 of the same admin user, unless there's something actually creating it (could be Flows, custom extensions, duplicated API calls if there's external client/frontend creating users).

**Reply by halwesit** [reply]
Actually, we have a front-end for this server. but no one has permission to access the administrator's account.

I've tried to remove the image from the account's avatar, but have not been able to save it.

------
![directus-2](https://user-images.githubusercontent.com/10852176/183090995-695e7402-0a0d-407c-a1b1-6f9d9ca34562.png)

**Reply by azrikahar** [reply]
Were you doing any import/export in users? That error sounds like it might be due to #6321, perhaps you can try whether the shared solution helps resolve that error: https://github.com/directus/directus/issues/6321#issuecomment-869349722

**Reply by halwesit** [reply]
I'm not able to run the SQL queries because I'm using caprover as a PaaS server for the Directus and Postgres sql.

**Reply by azrikahar** [reply]
I'm not exactly familiar with Caprover, but from what I understand, I'd assume it should still be possible to access the droplet (if using DigitalOcean one-click), or alternatively create an app like Adminer in Caprover to access your PostgreSQL db (assuming it's created in Caprover).

**Reply by halwesit** [reply]
I have used a custom VPS installed Caprover, but I'm able to install Adminer on it. so I can delete it throw Adminer. 

thanks for your help and time.  💯
